tests/test_cli_main.py

```diff
@@ -202,6 +214,142 @@ def test_handle_url_with_json_and_stream_url(self, console, args):
             self.assertEqual(console.exit.mock_calls, [call("The stream specified cannot be translated to a URL")])
             console.exit.mock_calls.clear()
 
+    def test_create_filename_invalid_chars_native_rules_auto(self):
+        streamlink_cli.main.args = args = Mock()
```

Comment:
Please properly mock this with the `@patch` decorator, so that it gets cleaned up after the test. Otherwise you're polluting the module's namespace.